test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -52,7 +60,7 @@ public TestClient(bool synchronousCallbacks = false)
                 Application.Out.TryWrite(memoryStream.ToArray());
             }
         }
-
+        
```

Comment:
It is a line containing whitespaces - if you are using VS you can install [this extension](https://marketplace.visualstudio.com/items?itemName=PawelKadluczka-MSFT.TrailingSpaceFlagger) and you will never miss it.